packages/shared/locales/en.json

```diff
@@ -1271,6 +1272,15 @@
         "refreshTokenMetadataHint" : "This will reset all your tokens' metadata including the verification status and unhide any hidden assets."
 
     },
+    "filters":{
+        "title": "Filters",
+        "assets": {
+            "showHidden": "Show hidden"
+        },
+        "activity": {
+            "showHidden": "Show hidden"
+        }
```

Comment:
Maybe this could be one locale only?

Reply:
there won't be any overlaps on the other filters that will come in the next branches. So i would add each filter for each filtertype on the corresponding locale object and therefore let this duplicate

Reply:
I would argue that it's generic for a filter to show hidden items, therefor at the moment we don't need the two branches (assets & activity). We can add branches when we clearly need to distinguish some filter locales based on filter type.